src/mixins/itext_behavior.mixin.ts

```diff
@@ -1,15 +1,12 @@
 // @ts-nocheck
 
 import { getEnv } from '../env';
-import { ObjectEvents, TEvent } from '../EventTypeDefs';
+import { ObjectEvents, TEvent, TPointerEvent } from '../EventTypeDefs';
 import { Point } from '../point.class';
 import { Text } from '../shapes/text.class';
-import { TPointerEvent } from '../typedefs';
```

Comment:
do we have 2 definitions or was this a broken import?

Reply:
broken import due to `@ts-nocheck`